### PR TITLE
States: add unavailable and unknown

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

States: add unavailable and unknown
- change state names to lowercase
- rephrase the statements "keeps track of entities in your environment" to no longer mention "environment"
- use bold to markup state name

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Release Notes

- **New Features**
	- Introduced new states **Unavailable** and **Unknown** for various entities, including air quality, alarm control panel, binary sensors, buttons, calendars, cameras, climate controls, covers, date/time entities, device trackers, events, fans, images, lawn mowers, locks, media players, notifications, numbers, remotes, scenes, selects, sirens, switches, to-do lists, text-to-speech, updates, vacuums, valves, wake word detection, and water heaters.
	- Enhanced state descriptions by standardizing formatting to lowercase for better consistency across integrations.

These additions enhance the clarity and functionality of the entities, providing users with better insights into their operational status.

### CODE REVIEW

File: source/_integrations/humidifier.markdown [hunk 1]
```diff
@@ -25,6 +25,11 @@ The **Humidifier** {% term integration %} is built for the controlling and monit
 
 The state of a humidifier entity can be either `on` or `off`.
```
Comment (klaasnicolaas) [inline source/_integrations/humidifier.markdown]:
```suggestion
The state of a humidifier entity can be either **on** or **off**.
```

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_integrations/alarm_control_panel.markdown [hunk 2]
```diff
@@ -25,16 +25,18 @@ Please see [manual alarm](/integrations/manual) or [template alarm](/integration
...
-- **Armed away**: The alarm is armed in away mode.
-- **Armed night**: The alarm is armed in night mode.
-- **Armed vacation**: The alarm is armed in vacation mode.
-- **Armed custom bypass**: The alarm is armed in bypass mode.
-- **Pending**: The alarm is pending (towards triggered).
-- **Arming**: The alarm is in the process of being armed.
-- **Disarming**: The alarm is in the process of being disarmed.
-- **Triggered**: The alarm has been triggered.
+- **disarmed**: The alarm is disarmed (off).
+- **armed home**: The alarm is armed in home mode.
```
Comment (MartinHjelmare) [inline source/_integrations/alarm_control_panel.markdown]: Should we write the exact string that is set, ie `"armed_home"` or is it enough to use the same name?

File: source/_integrations/switch.markdown [hunk 3]
```diff
@@ -18,7 +18,7 @@ related:
     title: Dashboard
 ---
 
-The **Switch** {% term integration %} keeps track of the states of the switches in your environment and allows you to control them.
+The **Switch** {% term integration %} manages the state of the switches and allows you to control them.
```
Comment (MartinHjelmare) [inline source/_integrations/switch.markdown]:
```suggestion
The **Switch** {% term integration %} manages the state of the switch entities and allows you to control them.
```

File: source/_integrations/alarm_control_panel.markdown (hunk 2 above)
Comment (c0ffeeca7) [reply source/_integrations/alarm_control_panel.markdown]: Well, I actually meant to use the UI string, not the technical name.
<https://github.com/home-assistant/core/blob/dev/homeassistant/components/alarm_control_panel/strings.json#L36>

**Review by MartinHjelmare — APPROVED** [review]

**Comment by c0ffeeca7** [comment]
Thank you all for the review!

**Pull request merged**